README.md

```diff
@@ -122,7 +122,23 @@ $ cd anbox
 $ mkdir build
 $ cd build
 $ cmake ..
-$ make
+$ make -j$(nproc)
+```
+#### cmake error: no cpu_features
```

Comment:
Can we move this to `docs/troubleshooting.md` and link here?

Reply:
Missed this. 

You could, but there's no real cost to having it in primary dependencies. 

at worse apt/pacman etc. just let's the user know it's already at the latest version, same as any other deps